core/src/main/scala/com/mesosphere/usi/core/protos/ProtoBuilders.scala

```diff
@@ -117,6 +118,7 @@ private[usi] object ProtoBuilders {
   def newTaskStatus(
       taskId: Mesos.TaskID,
       state: Mesos.TaskState,
+      uuid: ByteString, // UUID should be present, only e.g. TASK_ERROR does not have UUID
```

Comment:
Hm what do you think about taking a java.util.UUID class here and doing the serialisation inside the method? 

I'm asking because there are several ways to serialize UUID into bytes, such as `uuid.getBytes`, `uuid.toString.getBytes`, having a ByteString with 2 longs and so on. I'd prefer to do that in this method.

Reply:
well we are actually not the people creating these. UUID comes from mesos and we just put it into the ACK message. So I think we should ideally not touch it and keep it as close to pure bytes as possible.

Reply:
Keep in mind: this is the *Protobuf ByteString* holding the UUID and this method can only be used for mock testing

Reply:
Alright, no questions then :)